elements/pl-barcode-scan/pl-barcode-scan.py

```diff
@@ -0,0 +1,82 @@
+import prairielearn as pl
+import lxml.html
+import chevron
+
+BARCODE = '_pdf_barcode_scan'
+BLANK_ANSWER = ''
+REQUIRED = 'required'
+REQUIRED_DEFAULT = True
+
+
+def crc16arc(msg):
+    """
+    CRC-16/ARC
```

Comment:
There are many CRC-16 algorithms available. I had to deduce that this one is used in `js-crc`, as the checksum logic has to be consistent on the JavaScript front-end, Node-js back-end and Python back-e nd, but I did not find any documentation.